pkg/daemon/update.go

```diff
@@ -323,7 +323,7 @@ func ExtractOSImage(imgURL string) (osImageContentDir string, err error) {
 	}
 
 	// Extract the image
-	args := []string{"image", "extract", "--path", "/:" + osImageContentDir}
+	args := []string{"image", "extract", "-v", "10", "--path", "/:" + osImageContentDir}
```

Comment:
I will note that this function is only there for backward compatibility. By default all new rhel-coreos-8 images should not be hitting this legacy path.

That said, I assume you are only doing this for extensions below? Since that path should still be used. If you're not using extensions, may I ask how you're hitting this path?

I can't seem to find an example of what v10 logging looks like. The only way this could occur should be during an extensions test (since image extraction isn't hit). This test run actually failed during the extensions test https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_machine-config-operator/3493/pull-ci-openshift-machine-config-operator-master-e2e-gcp-op/1615087823960936448/artifacts/e2e-gcp-op/test/build-log.txt (global timeout) but there were no logs related to extensions... it's most likely the previous logs were wiped?

Would you be able to show what v10 logging looks like?

Reply:
I was not sure which one of them specifically were running as there are 2 that do extract, though why no to be able to debug both in case of failure?

Reply:
Right, sorry, to clarify, this code shouldn't be running at all for 4.12+ for base image extraction, since that uses layering paths.

Only extensions code should be trigger-able. I am just wondering if you are using extensions (the MC directive, i.e. https://github.com/openshift/machine-config-operator/blob/master/docs/MachineConfiguration.md#rhcos-extensions) since none of those should be used by default.

If you are not providing a specific extension, I was just wondering what paths we could have ran into this code in, since again, it should never execute if the cluster is 4.12/4.13

Reply:
4.11? 4.10? This what we see is failing